registry/README.md

```diff
@@ -118,66 +118,109 @@ Note: ```quilt login``` is its own test that the server is working.  You can now
 
 To browse the catalog using a web browser, enter this location into your web browser: http://localhost:3000
 
-# Run a Private Registry in AWS
-
-## AWS Resources
-Running a Quilt Registry in AWS requires the following services:
-* a Postgres database. You can find instructions for setting up Postgres in RDS here: [Create a Postres Database in RDS](https://aws.amazon.com/rds/postgresql/)
-* an S3 bucket
-* an EC2 instance to run the registry, authentication service and catalog. Add the following policies to the security group: Postgres, HTTP/HTTPS, SSH and a custom TCP rule to enable port 5000.
-* an Elastic Load Balancer (ELB) to terminate SSL connections to the registry and catalog
-
-Once the resources have been created, ssh into the EC2 instance and configure the environment and run the services via Docker. You can make it easier to connect to your registry by creating a new DNS record to point to the EC2 instance's external IP address (e.g., quilt.yourdomain.com).
-
+## Systems configuration
+Running a Quilt Registry services:
+* a Postgres database. [Create a Postres Database in Amazon RDS](https://aws.amazon.com/rds/postgresql/)
+* an S3 bucket and access keys
```

Comment:
Looks like you're dropping AWS specific language at the top, but keeping or maybe adding specific mentions of S3. If it's not meant to be AWS specific, it would be clearer to use more general terms for the storage and compute resources.